scout/server/links.py

```diff
@@ -373,7 +373,7 @@ def iarctp53(hgnc_symbol):
     if hgnc_symbol != "TP53":
         return None
 
-    link = "https://p53.iarc.fr/TP53GeneVariations.aspx"
+    link = "https://tp53.isb-cgc.org/"
```

Comment:
Ah, we never deep linked that one, since the site was, ahem, a little cumbersome. We could revisit that again, it looks a little cleaner, but still not exactly an open api. In another PR! 😁

Reply:
I've checked yesterday, but making the link is still cumbersome, so I gave up